### PR TITLE
Nghiep create product

### PR DESCRIPTION
**Pull request by hh-Nghiep**

### CODE REVIEW

**Comment by hh-Nghiep** [comment]
done create product

File: components/common/layout/layout.js [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 import Header from "../header/header";
 
+export const cartItemsContext = createContext();
+
 export default function Layout({ children }) {
+  const [cartItems, setCartItems] = useState([]);
+
+  const addToCart = (product) => {
+    setCartItems((prev) => [...prev, product]);
+    console.log(cartItems);
```
Comment (thaiphong-dev) [inline components/common/layout/layout.js]: delete all console.log

File: components/common/banner/banner.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  return (
+    <div className="sec-banner bg0 p-t-80 p-b-50">
+      <div className="container">
+        <div className="row">
+          <div className="col-md-6 col-xl-4 p-b-30 m-lr-auto">
+            {/* <!-- Block1 --> */}
+            <Link href="/shop" passHref>
+              <div className="block1 wrap-pic-w">
+                <Image src={iBanner01} alt="Image-BANNER" />
+
```
Comment (thaiphong-dev) [inline components/common/banner/banner.js]: HTML should not have space here, check and remove all

File: pages/_app.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import Layout from "../components/common/layout/layout";
 import "../styles/globals.css";
+import { useState } from "react";
```
Comment (thaiphong-dev) [inline pages/_app.js:3]: remove code not use

**Pull request merged**